app/controllers/admin/notes_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Admin
+  class NotesController < ApplicationController
+    def create
+      @submission = Submission.find(note_params[:submission_id])
+      note = Note.create(gravity_user_id: @current_user, **note_params)
```

Comment:
I think it should be `author`, not `gravity_user_id`?

```suggestion
      note = Note.create(author: @current_user, **note_params)
```

Reply:
`@current_user` is actually the gravity id though, not the instantiated object.

Reply:
That's odd... but let's keep the `gravity_user_id` then.